scripts/e2e_pipeline.sh

```diff
@@ -50,59 +50,69 @@ if [ -z "${ENV}" ]; then
-		echo "Please provide the STARKNET_SEPOLIA_ACCOUNT_ADDRESS environment variable"
+		echo "Please provide the STARKNET_SEPOLIA_ACCOUNT_ADDRESS environment variable."
 		exit 1
 	fi
 	if [ -z "${STARKNET_SEPOLIA_PRIVATE_KEY}" ]; then
-		echo "Please provide the STARKNET_SEPOLIA_PRIVATE_KEY environment variable"
+		echo "Please provide the STARKNET_SEPOLIA_PRIVATE_KEY environment variable."
 		exit 1
 	fi
 	if [ -z "${EVM_PRIVATE_KEY}" ]; then
-		echo "Please provide the EVM_PRIVATE_KEY environment variable"
+		echo "Please provide the EVM_PRIVATE_KEY environment variable."
 		exit 1
 	fi
-	SKIP="--ignore tests/end_to_end/L1L2Messaging --ignore tests/end_to_end/CairoPrecompiles --ignore tests/end_to_end/EvmPrecompiles --ignore tests/end_to_end/test_kakarot.py"
-elif [ "${ENV}" = "kakarot-staging" ]; then
+	SKIP="--ignore tests/end_to_end/L1L2Messaging --ignore tests/end_to_end/CairoPrecompiles -k 'not test_should_set_account_bytecode and not test_should_set_account_nonce and not test_should_upgrade_account_class and not test_should_raise_when_class_hash_is_not_declared and not test_should_upgrade_class_hash and not test_should_transfer_ownership and not test_should_return_transaction_count and not test_should_withdraw_all_eth'"
+elif [ "${ENV}" = "sepolia-staging" ]; then
 	export EVM_PRIVATE_KEY="0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
-	export KAKAROT_STAGING_RPC_URL="https://juno-kakarot-testnet-stage.karnot.xyz"
-	export KAKAROT_STAGING_ACCOUNT_ADDRESS="0x7ecf6cd45c32ce84812e660cc176cb8b4de2e7a6d5916fe326bf871466fbe02"
-	export WEB3_HTTP_PROVIDER_URI="https://kkrt-rpc-kakarot-testnet-stage.karnot.xyz"
-	if [ -z "${KAKAROT_STAGING_PRIVATE_KEY}" ]; then
-		echo "Please provide the KAKAROT_STAGING_PRIVATE_KEY environment variable. The private key should be loaded using gpg: gpg -r recipient@kakarot.org --decrypt path/to/encrypted/key.gpg"
+	export RPC_URL="https://juno-kakarot-sepolia.karnot.xyz/"
+	export RPC_NAME="starknet-sepolia-staging"
+	export CHECK_INTERVAL=1
+	export MAX_WAIT=50
+	export WEB3_HTTP_PROVIDER_URI="https://rpc-kakarot-sepolia-staging.karnot.xyz/"
+	if [ -z "${STARKNET_SEPOLIA_STAGING_ACCOUNT_ADDRESS}" ]; then
+		echo "Please provide the STARKNET_SEPOLIA_STAGING_ACCOUNT_ADDRESS environment variable."
+		exit 1
+	fi
+	if [ -z "${STARKNET_SEPOLIA_STAGING_PRIVATE_KEY}" ]; then
+		echo "Please provide the STARKNET_SEPOLIA_STAGING_PRIVATE_KEY environment variable."
 		exit 1
 	fi
 
-	SKIP="--ignore tests/end_to_end/L1L2Messaging"
+	SKIP="--ignore tests/end_to_end/L1L2Messaging --ignore tests/end_to_end/CairoPrecompiles -k 'not test_should_set_account_bytecode and not test_should_set_account_nonce and not test_should_upgrade_account_class and not test_should_raise_when_class_hash_is_not_declared and not test_should_upgrade_class_hash and not test_should_transfer_ownership and not test_should_return_transaction_count and not test_should_withdraw_all_eth'"
```

Comment:
why are we skipping?

Reply:
used to be ran without the multisig so skipped them. do you want to keep them knowing that failure of some test might transfer owner ship or upgrade contracts?

Reply:
so now that they work with multisig we can keep them?

you mean there is a case where the contract might have ownership transferred and rekt?